Simplenote/Classes/FileManager+Simplenote.swift

```diff
@@ -13,4 +13,29 @@ extension FileManager {
 
          return url
      }
+
+    /// URL for Simplenote's shared app group directory
+    ///
+    var sharedContainerURL: URL {
+        containerURL(forSecurityApplicationGroupIdentifier: Constants.sharedDirectoryDomain + Constants.groupIdentifier)!
+    }
+}
+
+private struct Constants {
+    static let defaultBundleIdentifier = "com.codality.NotationalFlow"
+    static let groupIdentifier = rootBundleIdentifier() ?? Constants.defaultBundleIdentifier
+    static let sharedDirectoryDomain = "group."
+
+    static func rootBundleIdentifier() -> String? {
```

Comment:
Like this much much better!  I have added this extension to bundle.  Since we have rebased this off develop the widget/intent extensions don't exist here yet, but I will add the APP_EXTENSION constant to them in the feature branch